modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -139,7 +139,7 @@ object Model {
     * Represents a queue with different levels of details. E.g. it could be a list of Ids
     * Or a list of fully hydrated SequenceViews
     */
-  case class SequencesQueue[T](queue: List[T])
+  case class SequencesQueue[T](conditions: Conditions, queue: List[T])
```

Comment:
>  Did it break the client?

The whole PR was about fixing what broke when adding the extra parameter. Fortunately, it was all caught at compilation time.

I vaguely remember discussing about squashing all global parameters in one parameter in the engine, so we'll to do something like that when adding the operator.